### PR TITLE
DatePicker: add test

### PR DESCRIPTION
**Pull request by QingWei-Li**
Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's [Contributing Guide](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.md).
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

**Comment by Leopoldthecoder** [comment]
注释掉的是走不到的代码？

**Comment by QingWei-Li** [comment]
注释掉的是暂时没什么用，估计是历史遗留代码，又不敢删 🌚

File: packages/date-picker/src/picker.vue [hunk 1]
```diff
@@ -484,9 +484,9 @@ export default {
       }
 
       this.$nextTick(() => {
-        if (this.type.indexOf('time') !== -1) {
-          this.picker.focusEditor && this.picker.focusEditor('hours');
-        }
+        // if (this.type.indexOf('time') !== -1) {
```
Comment (furybean) [inline packages/date-picker/src/picker.vue]: 可以删，这个是另一个风格的 TimePicker 才需要的。

File: packages/date-picker/src/panel/time-range.vue [hunk 2]
```diff
@@ -166,11 +166,11 @@
       ajustScrollTop() {
         this.$refs.minSpinner.ajustScrollTop();
         this.$refs.maxSpinner.ajustScrollTop();
-      },
-
-      focusEditor(val) {
-        return this.$refs.minSpinner.focusEditor(val);
       }
+
+      // focusEditor(val) {
```
Comment (furybean) [inline packages/date-picker/src/panel/time-range.vue]: 删。

**Comment by QingWei-Li** [comment]
Done

**Pull request merged**